metals/src/main/scala/scala/meta/internal/metals/codeactions/ActionableDiagnostic.scala

```diff
@@ -38,7 +38,8 @@ class ActionableDiagnostic() extends CodeAction {
       .getDiagnostics()
       .asScala
       .groupBy {
-        case ScalacDiagnostic.ScalaAction(textEdit) =>
+        case ScalacDiagnostic.ScalaAction(textEdit)
+            if textEdit.getNewText() != null && textEdit.getRange() != null =>
```

Comment:
Alternatively, this could be fixed by your PR @ckipp01 so let me know if you want to merge that first

Reply:
I think we can merge that in right away. I'm sort of waiting in various places to get stuff merged, so maybe that will help get the ball moving. I'll merge.